acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/ManualInterlockSigningAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.tests.signing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.teku.bls.BLS;
+import tech.pegasys.teku.bls.BLSSecretKey;
+import tech.pegasys.teku.bls.BLSSignature;
+import tech.pegasys.web3signer.core.service.http.ArtifactType;
+import tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.Eth2SigningRequestBody;
+import tech.pegasys.web3signer.core.signing.KeyType;
+import tech.pegasys.web3signer.dsl.utils.Eth2RequestUtils;
+import tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.response.Response;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled("Requires access to Interlock on USB Armory II")
+public class ManualInterlockSigningAcceptanceTest extends SigningAcceptanceTestBase {
+  private static final MetadataFileHelpers metadataFileHelpers = new MetadataFileHelpers();
+
+  // following keys are expected at /bls/key1.txt...key11.txt in USB Armory
+  private final List<BLSSecretKey> blsSecretKeys =
+      List.of(
```

Comment:
can we make a list of hex strings, then only have the BLSSecretKey.fromBytes(Bytes32.fromHexString(val)) just written once inside a loop?

Reply:
Also - this is overkill for an AT - the AT just needs to prove that we can load a single key from the interlock (ok, maybe 2) and sign with them - no need to do 11.
IntegrationTests etc. should prove the underlying capability - the AT is just required to prove we've wired it all together correct at the application...